scripts/portstat

```diff
@@ -337,6 +337,8 @@ class Portstat(object):
             print(table_as_json(table, header))
         else:
             print(tabulate(table, header, tablefmt='simple', stralign='right'))
+        if multi_asic.is_multi_asic(): 
+            print("\nReminder: Please execute 'show interface counters -d all' to include internal links\n")
```

Comment:
will this reminder show up if the command is already 'show interface counters -d all'?

Reply:
@tjchadaga yes, I tested on all `show int counters` combinations and it was printed. 
will update in description for the rest